### PR TITLE
Add 2+7+0 monitoring plugin

### LINKED ISSUE #23: EAR Monitoring 2+7+0
It's currently not possible to monitor in 7.1.2, which is the most widespread supported configuration by 3rd party plugins.

### PR DESCRIPTION
**Pull request by rsjtaylor**
Fixes #23

### CODE REVIEW

File: ear-production-suite/lib/include/speaker_setups.hpp [hunk 1]
```diff
@@ -160,10 +160,10 @@ static const std::vector<SpeakerSetup> SPEAKER_SETUPS = {
                     Speaker{"AC_00010002", "R", "M-030", -30.f, 0.f, Layer::middle, false, false},
                     Speaker{"AC_00010003", "C", "M+000", 0.f, 0.f, Layer::middle, false, false},
                     Speaker{"AC_00010004", "LFE", "LFE1", 45.f, -30.f, Layer::bottom, true, false},
-                    Speaker{"AC_0001000b", "SiR", "M-090", -90.f, 0.f, Layer::middle, false, false}, 
-                    Speaker{"AC_0001000c", "TpC", "T+000", 0.f, 90.f, Layer::upper, false, true}, 
-                    Speaker{"AC_0001001c", "LB", "M+135", 135.f, 0.f, Layer::middle, false, false}, 
-                    Speaker{"AC_0001001d", "Rrs", "M-135", -135.f, 0.f, Layer::middle, false, false}, 
+                    Speaker{"AC_0001000a", "SiL", "M+090", -90.f, 0.f, Layer::middle, false, false},
+                    Speaker{"AC_0001000b", "SiR", "M-090", 90.f,  0.f, Layer::middle, false, false},
```
Comment (firthm01) [inline ear-production-suite/lib/include/speaker_setups.hpp:164]: Are these +90, -90 polarities correct?

Comment (firthm01) [reply ear-production-suite/lib/include/speaker_setups.hpp:164]: Common def for AC_0001000a is +90.f, and for AC_0001000b is -90.f

**Review by firthm01 — CHANGES_REQUESTED** [review]
Just az polarities to double-check

File: ear-production-suite/lib/include/speaker_setups.hpp (hunk 1 above)
Comment (rsjtaylor) [reply ear-production-suite/lib/include/speaker_setups.hpp:164]: Good spot, thanks!

**Pull request merged**